app/gui/view/documentation/build.rs

```diff
@@ -0,0 +1,44 @@
+//! This script would run Tailwind CLI utility to generate a CSS stylesheet by scanning the
+//! source code for class names and including needed CSS rules in the output file.
+//!
+//! This script would only run if the `RUN_TAILWIND` environment variable is defined.
```

Comment:
Why? No information why such a design.

Reply:
I wrote about it here: https://github.com/enso-org/enso/pull/3999#issuecomment-1361110315
Do I need to add this info to the build script docs?